data/brands/shop/houseware.json

```diff
@@ -4,6 +4,17 @@
     "exclude": {"generic": ["^houseware$"]}
   },
   "items": [
+    {
+      "displayName": "House",
+      "locationSet": {"include": ["au"]},
+      "matchNames": ["House Bed & Bath"],
```

Comment:
```suggestion
```
Looking at https://www.house.com.au/store-locator, it seems like `House Bed & Bath` should have its own NSI entry unless that branding is being phased out.

Reply:
I do not think it is being phased out. I added it to the matchNames as the wikidata page has it under "Also Known As". In the store locator it is under the House brand. It looks like it is more specific for selling Bed and Bath linens and items. So I could see it being it's own entry. But at the same time in the store locator, if you click on all the drop down menus with all the stores in the different regions, there are only 13 of them. Which is barely over the inclusion limit. Also House Bed & Bath I think would still classify under the `shop=houseware` category.